A3A/addons/core/functions/Missions/fn_missionRequest.sqf

```diff
@@ -279,11 +282,33 @@ switch (_type) do {
 					if ([_site] call _checkRivalsTaskPossibility) then {
 						[[_site],"A3A_fnc_RIV_RES_Prisoners"] remoteExec ["A3A_fnc_scheduler",2];
 					} else {
-						[[_site],"A3A_fnc_RES_Prisoners"] remoteExec ["A3A_fnc_scheduler",2];
+						if (tierWar >= 5) then {
+							private _roll = round random 100;
+							if(_roll < 99) then {
+								[[_site],"A3A_fnc_RES_Deserters"] remoteExec ["A3A_fnc_scheduler",2];
+							} else {
+								[[_site],"A3A_fnc_RES_Prisoners"] remoteExec ["A3A_fnc_scheduler",2];
+							};
+						} else {
+							[[_site],"A3A_fnc_RES_Prisoners"] remoteExec ["A3A_fnc_scheduler",2];
+						};
 					};
 				};
 			};
-			if (_site in citiesX) then {[[_site],"A3A_fnc_RES_Refugees"] remoteExec ["A3A_fnc_scheduler",2]} else {[[_site],"A3A_fnc_RES_Prisoners"] remoteExec ["A3A_fnc_scheduler",2]};
+			/* if (_site in citiesX) then {
+				[[_site],"A3A_fnc_RES_Refugees"] remoteExec ["A3A_fnc_scheduler",2];
+			} else {
+				if (tierWar => 5) then {
+					private _roll = round random 100;
+					if(_roll < 50) then {
+						[[_site],"A3A_fnc_RES_Deserters"] remoteExec ["A3A_fnc_scheduler",2];
+					} else {
+						[[_site],"A3A_fnc_RES_Prisoners"] remoteExec ["A3A_fnc_scheduler",2];
+					};
+				} else {
+					[[_site],"A3A_fnc_RES_Prisoners"] remoteExec ["A3A_fnc_scheduler",2];
+				};
+			}; */
```

Comment:
No point having massive comments if they're uhh... pointless

Reply:
commented it out due to not knowing why this ```if (_site in citiesX) then {[[_site],"A3A_fnc_RES_Refugees"] remoteExec ["A3A_fnc_scheduler",2]} else {[[_site],"A3A_fnc_RES_Prisoners"] remoteExec ["A3A_fnc_scheduler",2]};``` was there

Reply:
If it's a city mission use refugees, if not use prisoners?